sources/p987.6/document.adoc

```diff
@@ -0,0 +1,36 @@
+= Draft Recommended Practice for Preparing an IEEE Standards Draft
+:docnumber: P987.6/D3
```

Comment:
@manuelfuenmayor
1. `docnumber` only includes 987.6
2. `3` is the `draft` value.

Reply:
Fixed.